recipes-core/distro-feed-configs/distro-feed-configs.bbappend

```diff
@@ -1,18 +1,18 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 
 inherit ptest
 
-SRC_URI_append = " \
-    file://run-ptest \
-    file://ptest-format.sh \
-    file://test_feed_listings.sh \
+SRC_URI:append = " \
+	file://run-ptest \
+	file://ptest-format.sh \
+	file://test_feed_listings.sh \
 "
 
-RDEPENDS_${PN}-ptest += " bash "
+RDEPENDS:${PN}-ptest += " bash "
 
-do_install_ptest() {
-    install -d ${D}${PTEST_PATH}
-    install -m 755 ${WORKDIR}/test_feed_listings.sh ${D}${PTEST_PATH}
-    install -m 644 ${WORKDIR}/ptest-format.sh ${D}${PTEST_PATH}
-    install -m 755 ${WORKDIR}/run-ptest ${D}${PTEST_PATH}
+do_install:ptest() {
```

Comment:
`do_install_ptest` isn't an override. [It's a task](https://github.com/ni/openembedded-core/blob/nilrt/master/hardknott/meta/classes/ptest.bbclass#L39) of the `ptest.bbclass`, and should use underscores.